task/sanity-label-check/0.1/sanity-label-check.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         #!/usr/bin/env bash
 
         . /utils.sh
-        if [ ! -f ../sanity-inspect-image/image_inspect.json ] || [ -z $(cat ../sanity-inspect-image/image_inspect.json) ]; then
+        if [[ ! -f ../sanity-inspect-image/image_inspect.json ]] || [[ -z $(cat ../sanity-inspect-image/image_inspect.json) ]]; then
```

Comment:
you can use '[ ! -s  ../sanity-inspect-image/image_inspect.json ]' `man test`:
```
       -s FILE
              FILE exists and has a size greater than zero
```

Reply:
Thanks, updated.